ui/src/app/store/pod/index.ts

```diff
@@ -0,0 +1,129 @@
+import { PayloadAction } from "@reduxjs/toolkit";
+
+import { createStandardSlice, GenericState } from "app/store/utils";
+import { Pod } from "./types";
+import { TSFixMe } from "app/base/types";
+
+export const DEFAULT_STATUSES = {
+  deleting: false,
+  refreshing: false,
+};
+
+const podSlice = createStandardSlice({
```

Comment:
Pod is a good candidate for this test as it requires additional state and reducers/actions.